### PR TITLE
Feature/crank stick

### PR DESCRIPTION
**Pull request by nferner**
This code will render movement for the crank-o-meter as well as render anger words dependent on the guest's score.

### CODE REVIEW

File: ReadME.md [hunk 1]
```diff
@@ -7,9 +7,26 @@ Please rank your crank based on the scenarios below where 1 is annoyed and 3 is
...
 ##**Color Palette**
-green       25B222 - crank-o-meter low crank (small sliver) have blend style into gold, send buttons 
+green       25B222 - crank-o-meter low crank (small sliver) have blend style into gold, send buttons
 gold        FFD700 - medium crank blend into red
 nose red    FF0B00 - high crank, logout button
 blue        212fff - score div background (white lettering)
 dot pink    FF9A9D - background for questions background for sign in inputs
 Wakko Blue  79CEFF - when editing sentences turn background of input box
+
+##Technologies Used##
```
Comment (arianacofone) [inline ReadME.md:17]: Thanks for starting the ReadMe!

File: client/components/guest/game-components/QuestionList.jsx [hunk 2]
```diff
@@ -20,10 +20,26 @@ class QuestionList extends Component {
     const clicked = values;
     const clientScore = this.state.clientScore;
     const total = (clientScore + clicked);
-    console.log(total);
     this.setState({ clientScore: total });
   }
+  componentDidUpdate() {
```
Comment (arianacofone) [inline client/components/guest/game-components/QuestionList.jsx:25]: Great job on this. I'm surprised you didn't lose your mind...

**Review by arianacofone — COMMENTED** [review]
Way to go!

File: ReadME.md (hunk 1 above)
Comment (tbongiorno1) [reply ReadME.md:17]: I second that!

File: client/components/guest/game-components/CrankOMeter.jsx [hunk 3]
```diff
@@ -1,14 +1,6 @@
 import React, { Component } from 'react';
 
 class CrankOMeter extends Component {
-  constructor (props) {
-    super(props);
-    this.answerChoiceOneHandler = this.answerChoiceOneHandler.bind(this);
-  }
-  answerChoiceOneHandler() {
-    const answerOneRotate = document.querySelector('#crankStick');
-    $('#crankStick').css({ transform: 'rotate(120deg + )'});
```
Comment (tbongiorno1) [inline client/components/guest/game-components/CrankOMeter.jsx:10]: great job on this.

File: client/components/guest/game-components/Score.jsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ class Score extends Component {
     return (
       <div id="score">
         <h1 id="scoreLogo"> Score </h1>
+        <p id="userScore"></p>
```
Comment (tbongiorno1) [inline client/components/guest/game-components/Score.jsx:11]: well done all around

**Pull request merged**